lapce-core/Cargo.toml

```diff
@@ -17,6 +17,7 @@ tree-sitter-javascript = "0.20.0"
 tree-sitter-typescript = "0.20.0"
 tree-sitter-python = "0.19.1"
 tree-sitter-toml = "0.20.0"
+tree-sitter-elixir = { git = "https://github.com/VitorTrin/tree-sitter-elixir.git"}
```

Comment:
Instead of using the fork, can you make a PR upstream at https://github.com/VitorTrin/tree-sitter-elixir with your changes?

Also, a `version` requirement (In this case I think the current release is  in the tree-sitter-elixer block couldn't hurt :) Otherwise we may explode the next time main / master gets pushed...

Cheers

Reply:
Done